deployment/k8s/charts/service-manager/values.yaml

```diff
@@ -0,0 +1,26 @@
+replicaCount: 1
+
+image:
+  repository: service-manager
```

Comment:
Ideally, there shouldn't be any difference between images for CF and k8s. That's why we have env variables/flags to point where it is supposed to run

Reply:
For now I will leave it with the local build. Once we organize the quay.io repo and how we will push in it it can be changed. For local development it could be easier to leave it like it is.